src/utils/DashboardHelperFunctions.js

```diff
@@ -68,23 +69,29 @@ export const applyEdits = (oktaAxios, formValues, incident) => {
   return putRequest;
 };
 
-// AddIncident.js
+// MapQuest API to get Latitude/Longitude used in clusters (Clusters.js)
 export const getLatAndLong = formValues => {
   const getRequest = new Promise((resolve, reject) => {
     const { city, state } = formValues;
+
+    if (!city || !state) {
+      throw new Error('Missing city or state');
+    } else {
+      city.toLowerCase();
+      state.toLowerCase();
+    }
+
+    const mapQuestURL = `https://open.mapquestapi.com/geocoding/v1/address?key=
+      ${process.env.REACT_APP_MAPQUEST_API_KEY}&location=${city},${state}`;
+
     axios
-      .get(
-        `https://open.mapquestapi.com/geocoding/v1/address?key=${
-          process.env.REACT_APP_MAPQUEST_API_KEY
-        }&location=${city ? city.toLowerCase() : ''}${
-          state ? ',' + state.toLowerCase() : ''
-        }`
-      )
+      .get(mapQuestURL)
```

Comment:
Also love the simplified `get` request by way of passing our completed URL to a variable, then that variable to the request!